diff --git a/package/opkg-utils/opkg-utils.mk b/package/opkg-utils/opkg-utils.mk
index ad267c5c94..384a7323ee 100644
--- a/buildroot/package/opkg-utils/opkg-utils.mk
+++ b/buildroot/package/opkg-utils/opkg-utils.mk
@@ -16,7 +16,7 @@ HOST_OPKG_UTILS_DEPENDENCIES = \
 	host-xz
 
 # Nothing to build; only scripts to install.
-define OPKG_UTILS_INSTALL_CMDS
+define OPKG_UTILS_INSTALL_TARGET_CMDS
 	$(MAKE) -C $(@D) PREFIX=$(TARGET_DIR) install-utils
 endef
 
