--- mosquitto-2.0.15.org/mosquitto.conf	2023-04-08 18:40:21.721373696 +0000
+++ mosquitto-2.0.15/mosquitto.conf	2023-04-08 18:40:22.189365507 +0000
@@ -231,7 +231,7 @@
 # listener 0 /tmp/mosquitto.sock
 #
 # listener port-number [ip address/host name/unix socket path]
-#listener
+listener 1883 0.0.0.0

 # By default, a listener will attempt to listen on all supported IP protocol
 # versions. If you do not have an IPv4 or IPv6 interface you may wish to
@@ -529,7 +529,7 @@
 # Defaults to false, unless there are no listeners defined in the configuration
 # file, in which case it is set to true, but connections are only allowed from
 # the local machine.
-#allow_anonymous false
+allow_anonymous true

 # -----------------------------------------------------------------
 # Default authentication and topic access control
