--- systemd-250.8.org/src/journal/journald.conf	2023-04-08 19:05:56.526129284 +0000
+++ systemd-250.8/src/journal/journald.conf	2023-04-08 19:06:57.456481393 +0000
@@ -15,14 +15,14 @@
 # See journald.conf(5) for details.

 [Journal]
-#Storage=auto
+Storage=volatile
 #Compress=yes
 #Seal=yes
 #SplitMode=uid
 #SyncIntervalSec=5m
 #RateLimitIntervalSec=30s
 #RateLimitBurst=10000
-#SystemMaxUse=
+SystemMaxUse=500M
 #SystemKeepFree=
 #SystemMaxFileSize=
 #SystemMaxFiles=100
