--- systemd-250.8.org/build/src/timesync/timesyncd.conf	2023-04-08 19:15:45.551363455 +0000
+++ systemd-250.8/build/src/timesync/timesyncd.conf	2023-04-08 19:17:17.128131994 +0000
@@ -14,7 +14,7 @@

 [Time]
 #NTP=
-#FallbackNTP=time1.google.com time2.google.com time3.google.com time4.google.com
+FallbackNTP=time.cloudflare.com
 #RootDistanceMaxSec=5
 #PollIntervalMinSec=32
 #PollIntervalMaxSec=2048
